### ISSUE TITLE
Affected by CVE-2024-21490 due to Angular Javascript inside ACS AEM Commons

### ISSUE THREAD

**Issue by henrykuijpers** [Adobe-Consulting-Services/acs-aem-commons#3291]
https://nvd.nist.gov/vuln/detail/CVE-2024-21490

"This package is EOL and will not receive any updates to address this issue. Users should migrate to [@angular/core](https://www.npmjs.com/package/@angular/core)."

I propose to either upgrade to the newest AngularJS, or simply remove it.

**Comment by kwin** [comment]
There is already lots of issues regarding outdated angular: https://github.com/Adobe-Consulting-Services/acs-aem-commons/issues?q=is%3Aissue+angular+is%3Aopen. Please rather comment on existing ones.

**Comment by henrykuijpers** [comment]
Yes, I saw that, but this is another new vulnerability with a score of 7.5. It needs to be fixed.

**Comment by davidjgonzalez** [comment]
@henrykuijpers  is this fixed in AngularJS 1.8.2?

**Comment by LucaNerlich** [comment]
> [@henrykuijpers](https://github.com/henrykuijpers) ~is this fixed in AngularJS 1.8.2?~ Ah - teh CVE is suggesting moving to Angular from AngularJS which are entirely different frameworks (the name is biggest similarity). Could you help rewrite the apps in ACS Commons that use AngularJS in another language (React/Angular/Svelte/whatever)?

Which apps inside acs-commons are this?
